typescript/libs/node_client.py

```diff
@@ -18,19 +18,34 @@
 
 
 class CommClient:
+    def serverStarted(self): pass
+
+    def workerStarted(self): pass
+
     def getEvent(self): pass
 
+    def getEventFromWorker(self): pass
+
     def postCmd(self, cmd): pass
 
+    def postCmdToWorker(self, cmd): pass
+    
     def sendCmd(self, cmd, cb): pass
 
+    def sendCmdToWorker(self, cmd, cb): pass
+
     def sendCmdSync(self, cmd): pass
 
+    def sendCmdToWorkerSync(self, cmd, cb): pass
+    
     def sendCmdAsync(self, cmd, cb): pass
 
+    def sendCmdToWorkerAsync(self, cmd, cb): pass
+
```

Comment:
This is debatable. So I've tried to make two subclasses of CommClient, which leads to the problem that our ``ServerProxy`` only takes one ``CommClient`` instance at a time. However some methods like ``